encoding/scenefiltering.md

```diff
@@ -61,16 +61,16 @@ import vapoursynth as vs
 core = vs.core
 
 src = core.lsmas.LWLibavSource("source.m2ts")
-filtered = core.resize.Bilinear(src, width=1280, height=720)
+resized = core.resize.Bilinear(src, width=1280, height=720)
 
-light_denoise = some_denoise_filter(filtered)
-heavy_denoise = some_other_denoise_filter(filtered)
+light_denoise = some_denoise_filter(resized)
+heavy_denoise = some_other_denoise_filter(resized)
 
-aa = antialiasing(filtered)
+aa = antialiasing(denoised)
```

Comment:
> # Creating the base filters
> Now that you have your common filtering down, you need to create some base filter chains. Go through some random scenes in your source and write down parts of the filtering that best suits those scenes.

I think it's clear that this part is just for creating exactly what is shown in this block, generic filters that we will use later on specific scenes.

i.e. if some scenes you looked through would be fixed with the func `antialiasing` here, you should write it down to choose what scenes need it later

Reply:
Hm, I guess you are right. Since I only read the diff, I didn't get the whole context on this page as I don't think I ever read it. I think we can make this clearer if you add a `denoised = …` line below the various denoised clips so that it becomes more obvious that something is missing here **and** `denoised` is actually defined. Probably do the samefor `debanded = …` below for consistency.